jobs/views.py

```diff
@@ -26,3 +29,20 @@ def job_details(request, id):
             'job': job,
         }
     )
+
+
+class JobCreate(CreateView):
+    model = Job
+    template_name = 'jobs/job_edit.html'
+    form_class = JobForm
+    success_url = reverse_lazy('jobs:jobs')
```

Comment:
We're going to want to redirect to a confirmation page with the link to donations etc - you can add a comment above this line that starts with "#TODO..." - in most text editors this will get highlighted a different colour and you can just use it as a reminder for things you're going to change later.